mihaela-etropolska/Homework 1/CV Mihaela Etropolska/mihaela-cv-style.css

```diff
@@ -0,0 +1,137 @@
+
+h1 {
+	text-align: center;
+	color: white;
+}
+
+p {
+	font-size: 30px;
+	color: white;
+}
+
+img {
```

Comment:
@mihaela-etropolska 
You probably want to reduce the image size, (400/500 looks good) otherwise the it will go outside the borders of your background. You probably want to find a background with greater height or add "background-attachment:fixed" : )